lm_eval/tasks/robustness/mmlu_pro/options_robustness_mmlu_pro.yaml

```diff
@@ -0,0 +1,26 @@
+task: options_robustness_mmlu_pro
+dataset_path: TIGER-Lab/MMLU-Pro
+dataset_name: default
+output_type: generate_until
+validation_split: validation
+test_split: test
+process_docs: !function utils.options_robustness_process_docs
+doc_to_text: !function utils.robustness_doc_to_text
+doc_to_target: answer
+generation_kwargs:
+  max_gen_toks: 1024
+  do_sample: False
+  until:
+    - "<|eot_id|>"
```

Comment:
this is model dependent, this is LLama's token, other models will have smth else